nixos/modules/programs/wayland/sway.nix

```diff
@@ -42,6 +42,11 @@ in {
       <https://github.com/swaywm/sway/wiki> and
       "man 5 sway" for more information'');
 
+    enableRealtime = mkEnableOption (lib.mdDoc ''
+      add CAP_SYS_NICE capability on `sway` binary for realtime scheduling
+      privileges. This may improve latency and reduce stuttering, specially in
+      high load scenarios'');
```

Comment:
Please add a comment that mentions that bwrap breaks with `Unexpected capabilities`

Reply:
I started `steam` and run a game, no issues in my system at least.

Reply:
CC @Emantor because of this comment: https://github.com/NixOS/nixpkgs/pull/203221.

Reply:
IMO the way better solution for this to work is to raise `rtprio` to one for every user. This way we don't need to mess with the capability handling.

Reply:
>  I think it's still not a good idea to give SYS_CAP_NICE to every process spawned by sway

Isn't raising `rtprio` even worse than?

Reply:
> > I think it's still not a good idea to give SYS_CAP_NICE to every process spawned by sway
> 
> Isn't raising `rtprio` even worse then?

Depends. raising rtprio to one only allows the lowest priority value for any process requesting real-time scheduling, while `CAP_SYS_NICE` allows any priority and scheduling class AFAICS.

Reply:
> @Emantor considering that you are the author of the PR in Sway that adds the SCHED_RR, could you give a patch to make it drop the capabilities? I can add it to the Sway and test.

Sure: https://gist.github.com/Emantor/5af76992f4e66befa3401bf96cf7fe97

Reply:
Had to modify the patch a little because it was not applying and also removed the log to keep it closer to upstream.